content/open-source/wolfi/building-a-wolfi-package.md

```diff
@@ -429,14 +450,15 @@ From this point, the process is essentially the following:
 - Create a branch with the name of your package, for instance: add-php-package
 - Remove the `repositories` and `keyring` sections of the YAML file
 - Add the `release-monitor` info to the YAML file
-- Add the package to the bottom of the `packages.txt` file in the root of the Wolfi OS repository
+- Run [`yam`](https://github.com/wolfi-dev/os/blob/main/docs/UPDATES.md) to fix any YAML formatting issues
 - Create a signed commit
 - Open a pull request following the instructions from the PR template.
 
-
-If you haven't yet, check the [Wolfi PHP package source file](https://github.com/wolfi-dev/os/blob/main/php.yaml) for a more comprehensive view of the melange YAML structure and how that looks in a more complex build.
+The [Wolfi Contributing Guide](https://github.com/wolfi-dev/os/blob/main/CONTRIBUTING.md) on GitHub has more details about this process.
 
 ## Resources to Learn More
 
+If you haven't yet, check the [Wolfi PHP package source file](https://github.com/wolfi-dev/os/blob/main/php.yaml) for a more comprehensive view of the melange YAML structure and how that looks in a more complex build.
```

Comment:
One more small change, this link does not work, should be:
https://github.com/wolfi-dev/os/blob/main/php-8.2.yaml
or 8.1, or 8.3, probably best to use 8.2 since it's the latest release.

Reply:
Since it's so late for you, let's get these GREAT improvements in, and I'll do a fast follow for the link fix.